test/image/compare_pixels_test.js

```diff
@@ -120,7 +110,7 @@ else {
 function untestableFilter(mockName) {
     var cond = !(
         mockName === 'font-wishlist' ||
-        mockName.indexOf('gl2d_') !== -1 ||
+        // mockName.indexOf('gl2d_') !== -1 ||
```

Comment:
:zap: this line completely.

Reply:
... and while at it can you :zap: the `gl2d` and `gl3d_cone` lines in comment above this function:

```js
/* Test cases:
 *
 * - font-wishlist
 * - all gl2d
 * - all mapbox
 * - gl3d_cone-*
 *
 * don't behave consistently from run-to-run and/or
 * machine-to-machine; skip over them for now.
 *
 */
```

Thanks!

Reply:
Done in be336489a93f6600c6ac668f4f7b46b323d99e53